### PR TITLE
Add IST0162 message description

### PR DESCRIPTION
**Pull request by nicole-lihui**
Please provide a description for what this PR is for.

Related problem: https://github.com/istio/istio/pull/45885

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [X] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by nicole-lihui** [comment]
The original PR is https://github.com/istio/istio/pull/45885

File: content/en/docs/reference/config/analysis/ist0162/index.md
Comment (hanxiaop) [inline content/en/docs/reference/config/analysis/ist0162/index.md:63]: I think `IngressGateway` should be consistent with line 8, which uses a different name.

File: content/en/docs/reference/config/analysis/ist0162/index.md
Comment (hanxiaop) [inline content/en/docs/reference/config/analysis/ist0162/index.md:66]: Should we specify the valid scenarios here to provide users with a better understanding?

**Comment by hanxiaop** [comment]
Let's wait until the istio repo's PR is merged. Add a hold label

File: content/en/docs/reference/config/analysis/ist0162/index.md
Comment (nicole-lihui) [reply content/en/docs/reference/config/analysis/ist0162/index.md:66]: nice 😄

**Comment by hanxiaop** [comment]
> This change is needed to allow the istio doc reference update to pass it's lint: #13918

I think this should be another problem where the message was not cleaned up, I'll submit a PR

**Comment by ericvn** [comment]
You likely need two things to make the lint work:
1) Remove the `content/zh/docs/reference/config/analysis/ist0104` directory which was missed in another PR
2) Run `make `update_ref_docs` to get the new `data/analysis.yaml` file that points to this message.

**Comment by ericvn** [comment]
This PR will then supersede https://github.com/istio/istio.io/pull/13918.

**Comment by ericvn** [comment]
> 1. Remove the `content/zh/docs/reference/config/analysis/ist0104` directory which was missed in another PR
> 2. Run `make update_ref_docs` to get the new `data/analysis.yaml` file that points to this message.

1. was done in another PR, so a rebase would pick that up.

**Review by hanxiaop — APPROVED** [review]

**Review by hanxiaop — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Comment by hanxiaop** [comment]
The istio repo's PR has been merged so remove DNM

**Pull request merged**
